agent/03_agent_build_installer.sh

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
-SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+DEVSCRIPTS_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 
-LOGDIR=${SCRIPTDIR}/logs
-source $SCRIPTDIR/logging.sh
-source $SCRIPTDIR/common.sh
-source $SCRIPTDIR/agent/common.sh
+LOGDIR=${DEVSCRIPTS_SCRIPT_DIR}/logs
+source $DEVSCRIPTS_SCRIPT_DIR/logging.sh
+source $SCRIDEVSCRIPTS_SCRIPT_DIRPTDIR/common.sh
```

Comment:
I tried running "make agent" to better understand #1403 and came across this error:
`
./agent/03_agent_build_installer.sh
+++ dirname ./agent/03_agent_build_installer.sh
++ cd ./agent/..
++ pwd
+ DEVSCRIPTS_SCRIPT_DIR=/home/rwsu/go/src/github.com/openshift-metal3/dev-scripts
+ LOGDIR=/home/rwsu/go/src/github.com/openshift-metal3/dev-scripts/logs
+ source /home/rwsu/go/src/github.com/openshift-metal3/dev-scripts/logging.sh
+++ dirname ./agent/03_agent_build_installer.sh
++ DEFAULT_LOGDIR=./agent/logs
++ LOGDIR=/home/rwsu/go/src/github.com/openshift-metal3/dev-scripts/logs
++ [[ -z '' ]]
+++ basename ./agent/03_agent_build_installer.sh .sh
+++ date +%F-%H%M%S
++ LOGFILE=/home/rwsu/go/src/github.com/openshift-metal3/dev-scripts/logs/03_agent_build_installer-2022-06-06-131804.log
+++ dirname /home/rwsu/go/src/github.com/openshift-metal3/dev-scripts/logs/03_agent_build_installer-2022-06-06-131804.log
++ '[' '!' -d /home/rwsu/go/src/github.com/openshift-metal3/dev-scripts/logs ']'
++ echo 'Logging to /home/rwsu/go/src/github.com/openshift-metal3/dev-scripts/logs/03_agent_build_installer-2022-06-06-131804.log'
Logging to /home/rwsu/go/src/github.com/openshift-metal3/dev-scripts/logs/03_agent_build_installer-2022-06-06-131804.log
++ exec
+++ tee /dev/fd/63
++++ awk '{ print strftime("%Y-%m-%d %H:%M:%S"), $0; fflush() }'
./agent/03_agent_build_installer.sh: line 8: SCRIDEVSCRIPTS_SCRIPT_DIRPTDIR: unbound variable
make: *** [Makefile:22: agent_build_installer] Error 1
`

Reply:
Ok I'll need to rebase and fix this error after 1403 is done. :+1: